x-pack/plugins/security_solution/public/cases/containers/mock.ts

```diff
@@ -203,7 +205,7 @@ export const basicCommentSnake: CommentResponse = {
 
 export const basicCaseSnake: CaseResponse = {
   ...basicCase,
-  status: 'open' as Status,
+  status: CaseStatuses.open as CaseStatus,
```

Comment:
I don't get a type error if I remove the typecast here. I think you're safe to get rid of it